lighthouse-core/test/audits/metrics/first-contentful-paint-3g-test.js

```diff
@@ -26,7 +26,7 @@ describe('Performance: first-contentful-paint-3g audit', () => {
 
     const result = await FCP3G.audit(artifacts, {settings: {}, options, computedCache: new Map()});
     // Use InlineSnapshot here so changes to Lantern coefficients can be easily updated en masse
```

Comment:
`:roll_eyes:` :P

Reply:
oh puh-lease 😛 there's like 40 freaking values to manually and copy and paste when lantern changes and I've had to do it like 8 times. do you want to volunteer to manually fix every expectation every time this happens from here on out?